src/com/lftechnology/java/training/krishna/employeemanagement/controllers/LoginController.java

```diff
@@ -0,0 +1,472 @@
+		return user;
+	}
+
+	/**
+	 * This method is used to display username
+	 *
+	 * @param username
+	 *            {@link String}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void showMainTitle(String username) {
+
+		System.out.println("\n" +
+			"*******************************************************" + "\n" +
+			"**                                                   **" + "\n" +
+			"**              Welcome To " + username + "                   **" +
+			"\n" + "**                                                   **" +
+			"\n" + "*******************************************************" +
+			"\n");
+	}
+
+	/**
+	 * This method is used to display main menu
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @param id
+	 *            {@link int}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void showMainMenu(Scanner scanner, int id) {
+
+		employeeServiceImpl = new EmployeeServiceImpl();
+		Employee employee = employeeServiceImpl.findById(id);
+		if (employee.getRole().equalsIgnoreCase(Role.Admin.toString())) {
+			System.out.println(""
+				+ "**   MAIN MENU (Please choose the desired action) *****"
+				+ "\n"
+				+ "**   1. Add New User                                 **"
+				+ "\n"
+				+ "**   2. Delete User                                  **"
+				+ "\n"
+				+ "**   3. Terminated User                              **"
+				+ "\n"
+				+ "**   4. Quit OR Logout                               **"
+				+ "\n"
+				+ "**   5. Search User                                  **"
+				+ "\n"
+				+ "*******************************************************"
+				+ "\n");
+
+			chooseAdminUserAction(scanner, id);
+		}
+		else {
+			System.out.println(""
+				+ "**   MAIN MENU (Please choose the desired action) *****"
+				+ "\n"
+				+ "**   4. Quit OR Logout                               **"
+				+ "\n"
+				+ "**   5. Search User                                  **"
+				+ "\n"
+				+ "**   6. Edit Information                             **"
+				+ "\n"
+				+ "*******************************************************"
+				+ "\n");
+
+			chooseNormalUserAction(scanner, id);
+		}
+
+	}
+
+	/**
+	 * This method is used to choose admin user action such as add employee,
+	 * delete user
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @param id
+	 *            {@link int}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void chooseAdminUserAction(Scanner scanner, int id) {
+
+		int choice = ValidationUtils.numberValidation(scanner);
+
+		switch (choice) {
+		case 1:
+			addEmployee(scanner);
+			break;
+		case 2:
+			deleteUser(scanner);
+			break;
+		case 3:
+			terminateUser(scanner);
+			break;
+		case 4:
+			authentiateUser(scanner);
+			break;
+		case 5:
+			searchEmployee(scanner);
+			break;
+		default:
+			System.out.println("Please choose the desired action.\n");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Please choose the desired action.\n" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)